src/script/conversation/MessageRepository.ts

```diff
@@ -1709,7 +1712,7 @@ export class MessageRepository {
     let messageType = eventInfoEntity.getType();
 
     if (messageType === GENERIC_MESSAGE_TYPE.CONFIRMATION) {
-      messageType += ` (type: "${eventInfoEntity.genericMessage.confirmation.type}")`;
+      messageType += ` (type: "${eventInfoEntity.genericMessage.confirmation?.type}")`;
```

Comment:
Will it be ok if `confirmation` will be `undefined`? I mean in this case it will produce:
```
 (type: undefined)
```

Reply:
ah nice catch, maybe it should cast to some value in that case

Reply:
It's no biggies since it's just a log, I didn't touch the logic at all in there. Theoretically it will never be undefined but if at some point it is, I think it's a good thing we see it in the log